t/api/08-jobtemplates.t

```diff
@@ -287,6 +288,12 @@ $res = $t->post_ok(
     })->status_is(200);
 my $job_template_id2 = $res->tx->res->json->{id};
 ok($job_template_id2, 'got ID (2)');
+is(
+    $app->schema->resultset('AuditEvents')
+      ->search({event => 'jobtemplate_create'}, {limit => 1, order_by => {-desc => 'id'}})->next->event_data,
+    to_json({id => $job_template_id2}),
```

Comment:
I'd rather use `is_deeply` and `decode_json` to compare Perl-hashes instead of JSON-strings.